BMM.Core/GuardedActions/TrackOptions/PrepareTrackOptionsAction.cs

```diff
@@ -60,6 +69,7 @@ protected override async Task<IList<StandardIconOptionPO>> Execute(IPrepareTrack
 
             var sourceVM = parameter.SourceVM;
             var track = (Track)parameter.Track;
+            bool shouldShowSleepTimerOption = sourceVM is PlayerViewModel && _featurePreviewPermission.IsFeaturePreviewEnabled();
```

Comment:
can we enable that through remote config?
So enable the sleep time if IsFeaturePreviewEnabled || RemoteConfig.SleepTimerEnabled

Reply:
done